### PR TITLE
Travis - CI testing

### PR DESCRIPTION
**Pull request by isuruf**

### CODE REVIEW

**Comment by certik** [comment]
Thanks! +1

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+ - 2.0
+ - 2.1
+
+install:
+  - export RUBY_SOURCE_DIR=`pwd`
+  - export TEST_CPP="no"
+
+  - git clone https://github.com/symengine/symengine symengine-cpp
+  - cd symengine-cpp
+  - git checkout 8537c67507fae9b1d8d575d322c6ade4f1a59297
```
Comment (abinashmeher999) [inline .travis.yml]: @isuruf I am unfamiliar with this type of command. What does this do?

Comment (certik) [inline .travis.yml]: There is a nice documentation about it here:

https://git-scm.com/docs/git-checkout

it checks out a given git commit.

Comment (abinashmeher999) [inline .travis.yml]: I see. Correct me if I am wrong. This seems to have the latest stable commit. I was wondering if we could use a tag. That would make this more clear, or a comment will also do.

Comment (certik) [inline .travis.yml]: Right. Eventually we'll use a release tag, but right now we are just trying to set it up.

**Pull request merged**